packages/docusaurus-theme-classic/src/theme/Footer/styles.module.css

```diff
@@ -13,3 +13,9 @@
 .footerLogoLink:hover {
   opacity: 1;
 }
+
+.iconExternalLink {
+  margin-left: 0.3rem;
+  position: relative;
+  top: 1px;
```

Comment:
why do we suddenly need these magic offsets that were not needed before?

Reply:
Those offsets where applied directly onto icon, I moved them to places where icons are actually used

Reply:
The styles are now applied on the consumer side instead of inlined in the SVG. Previously they lived in `IconExternal/styles.module.css`.

Reply:
```

import styles from './styles.module.css';

function IconExternalLink({width = 13.5, height = 13.5}: Props): JSX.Element {
  return (
    <svg
      width={width}
      height={height}
      aria-hidden="true"
      viewBox="0 0 24 24"
      className={styles.iconExternalLink}>
      <path
        fill="currentColor"
        d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"
      />
    </svg>
  );
}
```

Reply:
I see thanks 👍